src/metabase/api/search.clj

```diff
@@ -235,7 +163,7 @@
   (when query
     (into [:or]
           (for [column searchable-columns
-                token (search/tokenize (search/normalize query))]
+                token (scoring/tokenize (scoring/normalize query))]
```

Comment:
Arguably the NLP-related functions should go in their own namespace, but they're so trivial right now that I didn't bother

Reply:
`scoring/tokenize` can return an empty sequence--`(scoring/tokenize "\t")` for instance-- and this would yield an invalid query i think. Not sure if FE validation prevents searching for "  " or "\t" or other such things.

Reply:
> Not sure if FE validation prevents searching for " " or "\t" or other such things.

For now it does, but I'm sure this will be revisited as the FE continues to get polished